frontend/src/features/collection/state/collectionState.ts

```diff
@@ -57,9 +57,10 @@ export const collectionReducer = createReducer(initialState, {
     state.collectionLoading = true
   },
   'API_COLLECTION_SUCCESS': (state, action) => {
-    action.payload.data.forEach((workflow: WorkflowInfo) => {
-      state.collection[workflow.id] = workflow
-      state.ids.push(workflow.id)
+    console.log('success', action.payload.data)
+    action.payload.data.forEach((d: VersionInfo) => {
+      state.collection[d.initID] = d
+      state.ids.push(d.initID)
```

Comment:
I've written a bug here that I'd love you to fix: if the server sends back the same id multiple times (eg: when the filter / aggregation criteria changes) this `ids` array will contain duplicate IDs.

The right thing to do here is to remove the ids array from the state tree entirely & use `Object.keys(state.collection)` anywhere you need a list of IDs instead. Hopefully the only place we're relying on this array is within the `selectCollection` function within this file